packages/jest-repl/src/cli/repl.ts

```diff
@@ -5,30 +5,34 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  *
- * @flow
  */
 
-import type {GlobalConfig, ProjectConfig} from 'types/Config';
-
-declare var jestGlobalConfig: GlobalConfig;
-declare var jestProjectConfig: ProjectConfig;
-declare var jest: Object;
+declare var jestGlobalConfig: Config.GlobalConfig;
+declare var jestProjectConfig: Config.ProjectConfig;
 
 import path from 'path';
 import repl from 'repl';
 import vm from 'vm';
+import {Transformer} from '@jest/transform';
+import {Config} from '@jest/types';
 
-let transformer;
+let transformer: Transformer;
```

Comment:
I'm assuming transformer here is `Transformer` form `@jest/transform`

Reply:
Yes, it should uphold the same contract 🙂

Reply:
Cool beans.